gpytorch/settings.py

```diff
@@ -661,3 +662,16 @@ class use_toeplitz(_feature_flag):
     """
 
     _default = True
+
+
+class verbose(_feature_flag):
```

Comment:
If we target this to linalg ops, should we be more specific about the name? Like `verbose_linalg`?

Reply:
Will change.